pages/byoc/architecture.mdx

```diff
@@ -0,0 +1,35 @@
+# BYOC Architecture
+
+In this guide, we'll cover Bring Your Own Cloud(BYOC) feature in ModelZ, including its system architecture, agent, and security.
+
+## Architecture
+
+With BYOC, the data infrastructure, storage and network are owned by youself. We utilize a combination of your infrastructure and ModelZ hosted control plane service. You can deploy your machine learning models to any cluster (GCP, AWS, Lambda labs, your home lab, or even a single machine).There is no vendor lock and you can choose the most economical way to deploy models.
+
+![Architecture](./images/architecture.jpg)
+
+Let's see how it works:
+- Firstly, You can refer to the [install document](./installation.mdx), install agent components.
+- Once the agent is ready, developers can deploy models through the [ModelZ Console](../gettingstarted/deploy.mdx),[ModelZ CLI](../cli.mdx) or [ModelZ SDK](../sdk.mdx). 
+- The ModelZ control plane, deployed outside of your private environment, interacts with the agent in your cluster, ensuring accurate and real-time state synchronization and updates according to the configurations defined.
+- The data plane, deployed inside your private environment, resources related to the inference model will be created, such as deployment, service. If you want to expose your model deployment to the internet, it will create additional ingress for external access. 
```

Comment:
```suggestion
- The data plane, deployed within your private environment, creates resources related to the inference models, such as deployments and services. If you want to expose your model deployment to the internet, it will create additional ingress for external access. 
```